neurovault/apps/statmaps/views.py

```diff
@@ -1002,7 +1002,7 @@ def find_similar_json(request, pk, collection_cid=None):
 
     image1 = get_image(pk, None, request)
     pk = int(pk)
-    max_results = 100
+    max_results = Image.objects.all().count()  # image number will always be higher than comparison number
```

Comment:
This will result in the find_similar view showing all images instead of the first 100. Page will be huge and will load slow.